src/main/scala/org/renci/chemotext/Main.scala

```diff
@@ -52,7 +52,10 @@ object PubMedArticleWrapper {
     maybeYear.map { year =>
       // We can't parse a month-only date: to parse it, we need to include the year as well.
       val monthStr = (date \\ "Month").text
-      val maybeMonth: Try[Int] = Try(YearMonth.parse(s"$year-$monthStr", DateTimeFormatter.ofPattern("uuuu-MMM")).getMonth.getValue)
+      val maybeMonth: Try[Int] = Try({
+        if (monthStr.forall(Character.isDigit)) monthStr.toInt
+        else YearMonth.parse(s"$year-$monthStr", DateTimeFormatter.ofPattern("uuuu-MMM")).getMonth.getValue
+      })
```

Comment:
Somewhat simpler (is it the same result?):

```scala
val maybeMonth: Try[Int] = Try(monthStr.toInt)
  .orElse(Try(YearMonth.parse(s"$year-$monthStr", DateTimeFormatter.ofPattern("uuuu-MMM")).getMonth.getValue))
```

Reply:
Yes, that's much tidier! Fixed in fdc4b95.